src/main/java/seedu/dietbook/list/FoodList.java

```diff
@@ -136,7 +137,7 @@ public boolean clear() {
     /**
      * Obtain list of foods consumed within the range of a specified timing.
      */
-    public List<Food> getFoodsInDateTimeRange(LocalDateTime start, LocalDateTime end) {
+    public static List<Food> getFoodsInDateTimeRange(LocalDateTime start, LocalDateTime end) {
```

Comment:
I disagree with changing this to a static method. Instantiate the FoodList object instead (or get a pointer to the object via argument parameters) to call this method.